### DISCUSSION TITLE
The API has been down for a few days.

### DISCUSSION THREAD

**Discussion by arjunkahlon** [gadhagod/Hyrule-Compendium-API#37] [Q&A] [answered]
Hi, the API has been down for a few days. Do you know when it will be back up?

**Comment by gadhagod** [comment, marked as answer]
Sorry about this!
It will be back up in 1-2 days. 
Thanks for letting me know.

**Reply by gadhagod** [reply]
The API is back up. I apologize for the inconvenience.

https://github.com/gadhagod/Hyrule-Compendium-API/actions/runs/3326134232

**Reply by arjunkahlon** [reply]
Much appreciated. Thank you.
